src/main/java/blackjack/BlackJackGame.java

```diff
@@ -0,0 +1,90 @@
+package blackjack;
+
+import blackjack.card.CardDeck;
+import blackjack.card.Hand;
+import blackjack.card.initializer.StandardCardsInitializer;
+import blackjack.gamer.Betting;
+import blackjack.gamer.Dealer;
+import blackjack.gamer.Nickname;
+import blackjack.gamer.Player;
+import java.util.ArrayList;
+import java.util.List;
+import view.InputView;
+import view.OutputView;
+
+public class BlackJackGame {
+
+    private static final String YES_SIGN = "y";
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public BlackJackGame(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void start() {
+        CardDeck deck = new CardDeck(new StandardCardsInitializer());
+
+        Dealer dealer = new Dealer(new Hand(deck));
+        List<Player> players = setPlayers(deck);
+
+        outputView.printInitialCards(dealer, players);
+
+        processPlayersTurn(players, deck);
+
+        processDealerTurn(dealer, deck);
+
+        outputView.printCardResult(dealer, players);
+
+        outputView.printProfitResult(dealer.calculateProfits(players));
+    }
+
+    private List<Player> setPlayers(CardDeck deck) {
+        List<Player> players = new ArrayList<>();
+
+        List<Nickname> nicknames = inputView.readNames().stream()
+                .map(Nickname::new)
+                .toList();
+
+        for (Nickname nickname : nicknames) {
+            Betting betting = new Betting(inputView.readBettingAmount(nickname));
+            players.add(new Player(new Hand(deck), nickname, betting));
+        }
+
+        return players;
+    }
```

Comment:
아래와 같이 한번에 players를 생성할 수도 있을 것 같아요. 😃 

```suggestion
    private List<Player> setPlayers(CardDeck deck) {
        return inputView.readNames().stream()
            .map(nickName -> new Player(
                new Hand(deck),
                new Nickname(nickName),
                new Betting(inputView.readBettingAmount(nickName))
            ))
            .toList();
    }
```

Reply:
아.. 이렇게하면 닉네임 검증이 먼저 진행되어야하는데, 한명한명 `닉네임 검증 + 베팅금액 입력` 절차를 진행해버리겠군요? 🤔
이렇게 의도하신게 아니라면 제가 제안드린 방식은 아마 사용하지 못할 것 같습니다. 😅  참고정도만 해주세요. 

이 코멘트를 달면서 테스트해보다가 알게된 부분인데,  `InputView.readNames()` 메서드에 대해서    `로키,` 또는 `로키, ` 이런식으로 입력하게되면, NickName 객체 생성 시에 유효성 검증에 걸려 에러가 발생할 줄 알았는데, 정상적으로 게임을 수행하는군요...? 🤔  

이 부분은 의도하신 바가 아닐 것 같은데, 확인한번 부탁드립니다. 🙏

Reply:
꼼꼼히 확인해주셔서 감사합니다! 말씀해주신 것 처럼 해당 상황은 의도한 바가 아니었습니다.
InputView에 입력 유효성 검사 로직을 추가하여 보완하였습니다!